### DISCUSSION TITLE
Windows Key being disabled

### DISCUSSION THREAD

**Discussion by tuobsessed** [SnosMe/awakened-poe-trade#749] [Q&A] [answered]
Iv been trouble shooting for hours to figure out why my windows key is not working. Determined that the latest update of APOE disables the windows key of your keyboard. The problem even persists when closing out of the program. The only way iv been able to remedy it is be restarting my entire computer.

any ideas on a fix? i would like to keep using the utility

**Comment by SnosMe** [comment, marked as answer]
I don't interact with Windows keys in any way.

> latest update of APOE disables

Just a coincidence.

**Reply by tuobsessed** [reply]
i find it not to be a coincidence when i do not launch poetrade and my windows key works. after launching the software, it no longer works. there may be an unintended interaction. i can repeat this multiple times with the same result. 

****EDIT**** You are correct, I apologize. There was another hidden program that was disabling my windows key on POE start up. Which happened to be the same time I launch APOET. Thank you for your assistance.

**Comment by marquis51** [comment]
Hi, Im confused I have the same issue when I launch POE with awakened poe trade it disables my F keys, f1 to f12, how can I troubleshoot this ?

**Reply by marquis51** [reply]
So the weird behavior seems to have been caused by Ghub in my case, strangely only in POE. uninstalling it has solved the problem so far.
